parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -402,6 +402,17 @@ write_files:
     IFS=' '
     {{end}}
 {{end}}
+
+    # Disallow container from reaching out to the special IP address 168.63.129.16
+    # for TCP protocol (which http uses)
+    #
+    # 168.63.129.16 contains protected settings that have priviledged info.
+    #
+    # The host can still reach 168.63.129.16 because it goes through the OUTPUT chain, not FORWARD.
+    #
+    # Note: we shouldn't block all traffic to 168.63.129.16. For example UDP traffic is still needed
+    # for DNS.
+    iptables -I FORWARD -d 168.63.129.16 -p tcp -j DROP
```

Comment:
Why not only block the remote port 80?

Reply:
No specific reason. Blocked all ports since I don't know any useful use case that needs access to tcp 168.63.129.16 from container. Is Windows only blocking port 80?

Reply:
Windows only blocks the port 80 which Wire server is using. I raised this question because I also want to know whether you know some specific reason to block all ports :)

Reply:
Blocking all port seems safer than just blocking 80?